constants/src/lib.rs

```diff
@@ -1,5 +1,18 @@
 #![doc = include_str!("../README.md")]
-#![deny(missing_docs, reason = "all constants should document what they are")]
+#![forbid(
+    clippy::missing_assert_message,
+    clippy::missing_docs_in_private_items,
+    clippy::missing_errors_doc,
+    clippy::missing_panics_doc,
+    clippy::should_panic_without_expect,
+    clippy::single_char_lifetime_names,
+    missing_docs,
+    unsafe_code,
+    unused_results,
+    missing_copy_implementations,
+    missing_debug_implementations,
+    reason = "Crate-specific lints. There should be good reasoning when removing these."
+)]
```

Comment:
This is the full list of lints, all other crates have some subset of this. I will list them: